src/gtk/window.cpp

```diff
@@ -2460,7 +2460,7 @@ wxWindow *wxGetActiveWindow()
 
 // Under Unix this is implemented using X11 functions in utilsx11.cpp but we
 // need to have this function under Windows too, so provide at least a stub.
-#ifndef GDK_WINDOWING_X11
+#if !defined(GDK_WINDOWING_X11) && defined(__WXMSW__)
```

Comment:
I don't think `__WXMSW__` is ever defined when building wxGTK.
```suggestion
#ifdef GDK_WINDOWING_WIN32
```